HollowKnightSplitSettings.cs

```diff
@@ -739,6 +739,16 @@ public enum SplitName {
         EnterBeastDen,
         [Description("Blue Lake (Transition)"), ToolTip("Splits on transition to Blue Lake from either side")]
         BlueLake,
+        [Description("Bretta House (Transition)"), ToolTip("Splits on transition to Bretta's House")]
+        BrettaHouse,
+        [Description("Bretta House Bubbles (Transition)"), ToolTip("Splits on transition to the fourth room of Bretta's House C-Side")]
+        BrettaHouseBubbles,
+        [Description("Bretta House Bumpers (Transition)"), ToolTip("Splits on transition to the third room of Bretta's House C-Side")]
+        BrettaHouseBumpers,
+        [Description("Bretta House Switches (Transition)"), ToolTip("Splits on transition to the second room of Bretta's House C-Side")]
+        BrettaHouseSwitches,
+        [Description("Bretta House Zippers (Transition)"), ToolTip("Splits on transition to the first room of Bretta's House C-Side")]
+        BrettaHouseZippers,

```

Comment:
The Description text for the modded transitions should indicate that they are for Modded only, and the ToolTip text should specify the name of the mod.

Something like `Modded Bretta House ...` and `... of Bretta's House C-Side from the Scattered and Lost mod`, perhaps.